translations/ko/intro/describing-quantum-computers.ipynb

```diff
@@ -0,0 +1,674 @@
+      "source": [
+        "# 양자 컴퓨터 설명\n",
+        "\n",
+        "이 장에서는 양자컴퓨터를 설명하기 위해 사용하는 다양한 수학적 오브젝트와 표기법을 소개합니다. 수학의 기호나 방정식, 전문적인 어휘는 간결하고 확실한 이해를 도와주지만 기호의 의미를 알기 전에는 이해하기 힘들기 때문에 많은 사람들을 겁먹게 하곤 합니다. 그래서 이 챕터에서는 수식을 인터랙티브하게 보여줍니다. 기호 위에 마우스를 올려놓으면 그 기호가 무엇을 의미하는지 알 수 있습니다. 또 수학이나 양자컴퓨터의 언어를 구사할 수 있게 하기 위해서 [난해한 ](gloss:esoteric)단어를 조금씩 배치하였으며 이 단어에 마우스를 올려놓으면 그 단어의 해설을 볼 수 있게 되어 있습니다.\n",
+        "\n",
+        "## 진폭\n",
+        "\n",
+        "고전적인 확률은 0에서 1 사이의 [실수](gloss:real-number)로 나타나는 경우가 많지만 진폭은 방향도 가지고 있습니다. 진폭을 표현하기 위한 가장 훌륭한 방법은 [복소수](gloss:complex-number)입니다. 복소수는 크기와 방향 모두 완전히 기술할 수 있지만 이 과정에서는 두가지 방향(예를 들어 왼쪽과 오른쪽)을 가리키는 진폭만 다루고 그 외에는 신경쓰지 않기로 합니다.\n",
+        "\n",
+        "![진폭과 확률을 비교하는 이미지](images/quantum-states/prob-vs-amp.svg)\n",
+        "\n",
+        "이것으로 계산이 매우 간단해집니다. 즉 진폭을 -1부터 +1까지의 숫자로 나타낼 수 있고, 그 숫자가 양수이면 진폭은 앞을 향하고, 음수이면 뒤를 향하고 있기 때문입니다. 이 특성만으로도 재미있는 것들을 해볼 수 있습니다!\n",
+        "\n",
+        "<!-- ::: q-block.exercise -->\n",
+        "\n",
+        "### 짧은 퀴즈\n",
+        "\n",
+        "<!-- ::: q-quiz(goal=\"intro-describing-0\") -->\n",
+        "\n",
+        "<!-- ::: .question -->\n",
+        "\n",
+        "다음 중 확률값은 *아니지만* 진폭에 해당되는 값은 무엇입니까?\n",
+        "\n",
+        "<!-- ::: -->\n",
+        "\n",
+        "<!-- ::: .option(correct) -->\n",
+        "\n",
+        "1. $-1$\n",
+        "\n",
+        "<!-- ::: -->\n",
+        "\n",
+        "<!-- ::: .option -->\n",
+        "\n",
+        "1. $1/3$\n",
+        "\n",
+        "<!-- ::: -->\n",
+        "\n",
+        "<!-- ::: .option -->\n",
+        "\n",
+        "1. $1.01$\n",
+        "\n",
+        "<!-- ::: -->\n",
+        "\n",
+        "<!-- ::: .option -->\n",
+        "\n",
+        "1. $\\sqrt{-2}$\n",
+        "\n",
+        "<!-- ::: -->\n",
+        "\n",
+        "<!-- ::: -->\n",
+        "\n",
+        "<!-- ::: -->"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "formulas": {
+          "_amp-0-0": {
+            "meaning": "This is the amplitude of the two-qubit state <code>00</code>, it's equal to the square-root of a half."
+          },
+          "_amp-1-0": {
+            "meaning": "This is the amplitude of the two-qubit state <code>01</code>, it's equal to the square-root of a half."
+          },
+          "_amp-2-0": {
+            "meaning": "This is the amplitude of the two-qubit state <code>10</code>, it's equal to 0."
+          },
+          "_amp-3-0": {
+            "meaning": "This is the amplitude of the two-qubit state <code>11</code>, it's equal to 0."
+          },
+          "def-equal": {
+            "meaning": "This means we are defining the things on the left of this symbol to be equal to the things on the right of this symbol."
+          },
+          "x-ket": {
+            "meaning": "This is a vector we have labelled '|x⟩', the '|' and '⟩' remind us this label refers to a vector.",
+            "say": "ket x"
+          }
+        }
```

Comment:
this notebook has some metadata (`"formulas"`) which should be translated (i.e., the values of `"meaning"` and `"say"`)